src/lint.cpp

```diff
@@ -421,8 +433,15 @@ void linter::visit_end_of_module() {
        global_scope.variables_used_in_descendant_scope) {
     if (!is_variable_declared(used_var)) {
       // TODO(strager): Should we check used_var.kind?
-      this->error_reporter_->report(
-          error_use_of_undeclared_variable{used_var.name});
+      switch (used_var.kind) {
```

Comment:
Why did this code change? I'm confused.

Reply:
It was one of the first changes, it was for testing, but it doesn't affect anything so I went back to how it was.